docs/model-serving/storage/multiple-storage-uris.md

```diff
@@ -0,0 +1,430 @@
+        # Downloads to /mnt/models (default)
+```
+
+</TabItem>
+<TabItem value="mixed-paths" label="Mixed Paths">
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: multi-storage-mixed
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: pytorch
+    storageUris:
+      - uri: s3://bucket/base-model
+        path: /mnt/models/base
+      - uri: s3://bucket/preprocessing
+        # Uses default /mnt/models
+```
+
+</TabItem>
+</Tabs>
+
+Apply the InferenceService:
+```bash
+kubectl apply -f multi-storage.yaml
+```
+
+## Path Configuration
+
+### Default Behavior
+If no path is specified, models download to `/mnt/models`:
+
+```yaml
+storageUris:
+  - uri: s3://bucket/model
+    # Downloads to /mnt/models (default)
+```
+
+### Custom Paths
+Specify explicit mount paths for downloaded artifacts:
+
+```yaml
+storageUris:
+  - uri: s3://bucket/base-model
+    path: /mnt/models/base
+  - uri: s3://bucket/preprocessing
+    path: /mnt/models/preprocessing
+```
+
+## Path Requirements
+
+### Common Root Directory
+All custom paths must share a common root directory (excluding filesystem root):
+
+<Tabs>
+<TabItem value="valid" label="✅ Valid">
+
+```yaml
+# Common root: /mnt/models
+storageUris:
+  - uri: s3://bucket/model-a
+    path: /mnt/models/model-a
+  - uri: s3://bucket/model-b
+    path: /mnt/models/model-b
+```
+
+</TabItem>
+<TabItem value="invalid" label="❌ Invalid">
+
+```yaml
+# Different roots
+storageUris:
+  - uri: s3://bucket/model-a
+    path: /models/model-a     # Root: /models
+  - uri: s3://bucket/model-b
+    path: /other/model-b      # Root: /other
+```
+
+</TabItem>
+</Tabs>
+
+### Absolute Paths
+All custom paths must be absolute paths:
+
+```yaml
+# ✅ Valid
+path: /mnt/models/custom
+
+# ❌ Invalid
+path: models/custom
+```
+
+## Compatibility and Migration
+
+### Mutual Exclusivity
+The `storageUri` and `storageUris` properties are mutually exclusive:
+
+<Tabs>
+<TabItem value="valid-migration" label="✅ Valid">
+
+```yaml
+# Use either storageUri
+predictor:
+  model:
+    storageUri: s3://bucket/model
+
+# OR storageUris
+predictor:
+  model:
+    storageUris:
+      - uri: s3://bucket/model
+```
+
+</TabItem>
+<TabItem value="invalid-both" label="❌ Invalid">
+
+```yaml
+# Cannot use both
+predictor:
+  model:
+    storageUri: s3://bucket/model
+    storageUris:
+      - uri: s3://bucket/other-model
+```
+
+</TabItem>
+</Tabs>
+
+### Equivalent Configurations
+These configurations are functionally equivalent:
+
+<Tabs>
+<TabItem value="legacy" label="Legacy Single URI">
+
+```yaml
+storageUri: s3://bucket/model
+```
+
+</TabItem>
+<TabItem value="new-default" label="New with Default Path">
+
+```yaml
+storageUris:
+  - uri: s3://bucket/model
+```
+
+</TabItem>
+<TabItem value="new-explicit" label="New with Explicit Path">
+
+```yaml
+storageUris:
+  - uri: s3://bucket/model
+    path: /mnt/models
+```
+
+</TabItem>
+</Tabs>
+
+## Advanced Examples
```

Comment:
How is this different than the simple use case before? Is it because of the Lora adapters? Essentially, it is the same thing regarding configuration, no?